client/data/post-formats/use-post-formats-query.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { useQuery } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import wpcom from 'calypso/lib/wp';
+
+const usePostFormatsQuery = ( siteId ) =>
+	useQuery( [ 'postFormats', siteId ], () =>
+		wpcom.undocumented().site( siteId ).postFormatsList()
```

Comment:
I think we started using `wpcom.req.get` directly so we can eventually move away from `wpcom.undocumented`.

Reply:
Sure thing! Done in fa174d5bc515b11810f2d9d1a09f91731dfb8ddf. I've also removed the unused method in fa174d5bc515b11810f2d9d1a09f91731dfb8ddf.